Microsoft.Toolkit.Uwp.UI.Media/Geometry/Utils.cs

```diff
@@ -0,0 +1,984 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Numerics;
+using System.Runtime.InteropServices;
+using Windows.Foundation;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Geometry
+{
+    /// <summary>
+    /// Class containing collection of useful methods
+    /// for various types
+    /// </summary>
+    public static class Utils
```

Comment:
Think this should be internal? Otherwise we would want to think about adding them to `Microsoft.Toolkit`?

Reply:
Similar with the Scalar helpers as well. @Sergio0694 thoughts?

Reply:
Moved the Scalar class to Microsoft.Toolkit.
I haven't moved the Utils class to Microsoft.Toolkit as Utils uses Windows.Foundation.Point.

Reply:
With #3639 we have the core Toolkit libraries referenced by this package, so things can live in `Microsoft.Toolkit.Uwp` too. I believe we have some Color Helpers and extensions for Vectors/Sizes, etc... there already, so would be good to audit what can be consolidated with helpers we already have?